### DISCUSSION TITLE
Recommended way to typeset units?

### DISCUSSION THREAD

**Discussion by Qyriad** [typst/typst#1601] [Q&A]
LaTeX has the siunitx package, and MathJAX has `\pu`, but I don't see any obvious equivalents in the Typst documentation, and with #274 I can't use normal strings to typeset single-letter units without the hack of using an extra space. What is the recommended way to do this in Typst?

**Comment by fenjalien** [comment]
Just remembered I made a gist for this a while back. Give me a bit and I'll setup a proper repo (and maybe submit it as a package) for it!

**Reply by fenjalien** [reply]
Here we go: https://github.com/fenjalien/typst-units. Sorry theres no documentation in the readme yet but I need sleep.

As an example:
```
$ 1 unit(J / mol K) $
```
![image](https://github.com/typst/typst/assets/34489450/2acc84b6-f887-4981-88b9-30a5416679e5)
